kit/layout/src/Skeleton/Skeleton.tsx

```diff
@@ -213,7 +222,13 @@ export function UISkeleton({
 const styles = StyleSheet.create({
     container: {
         position: 'relative',
+    },
+    skeletonContainer: {
         overflow: 'hidden',
+        top: -StyleSheet.hairlineWidth,
+        left: -StyleSheet.hairlineWidth,
+        bottom: -StyleSheet.hairlineWidth,
+        right: -StyleSheet.hairlineWidth,
```

Comment:
That looks weird to me... Can you add a comment why do we need it?
Also, it looks like it's better to add `position: 'absolute'` here in order to ensure that the applied style contains it (without relying on `StyleSheet.absoluteFill`)